File: src/main/java/com/wanderers/hotelier_webservices/HotelierWebservicesApplication.java
package com.wanderers.hotelier_webservices;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.context.annotation.ComponentScan;
import org.springframework.context.annotation.FullyQualifiedAnnotationBeanNameGenerator;

@SpringBootApplication(
		nameGenerator = FullyQualifiedAnnotationBeanNameGenerator.class
)
@ComponentScan(
		basePackages = {"com.wanderers.hotelier_webservices.rest", "com.wanderers.hotelier_webservices.server", "com.wanderers.hotelier_webservices.config", "com.wanderers.hotelier_webservices.mapper"},
		nameGenerator = FullyQualifiedAnnotationBeanNameGenerator.class
)
public class HotelierWebservicesApplication {

	public static void main(String[] args) {
		SpringApplication.run(HotelierWebservicesApplication.class, args);
	}

}

File: src/main/kotlin/com/wanderers/hotelier_webservices/HotelierWebservicesApplication.kt
package com.wanderers.hotelier_webservices;

import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.runApplication
import org.springframework.context.annotation.ComponentScan
import org.springframework.context.annotation.FullyQualifiedAnnotationBeanNameGenerator

@ComponentScan(	basePackages = ["com.wanderers.hotelier_webservices.rest", "com.wanderers.hotelier_webservices.server", "com.wanderers.hotelier_webservices.config"])
@SpringBootApplication(nameGenerator = FullyQualifiedAnnotationBeanNameGenerator::class)
class HotelierWebservicesApplication

fun main(args: Array<String>) {
	runApplication<HotelierWebservicesApplication>(*args)
}